cfgov/legacy/templates/front/base_update.html

```diff
@@ -211,16 +204,6 @@
   //]]>
 </script>
 
-<script type="text/javascript">
-    // If there are no breadcrumbs on this page, remove the ul tag so that screen
-    // readers don't read it (since the ul has nothing in it).
-    $(function(){
-        var numBreadCrumbs = $( 'ul.bread' ).children( 'li' ).length;
-        if( numBreadCrumbs === 0 ) {
-           $( 'ul.bread' ).remove();
-        }
-    });
-</script>
```

Comment:
You can also remove the `<ul class="bread meta">` from this template, as nothing that extends this template puts anything in `{% block breadcrumbs %}`.